understanding/20/change-on-request.html

```diff
@@ -10,12 +10,13 @@ <h1>Understanding Change on Request</h1>
       
 		<section id="brief">
 			<h2>In brief</h2>
-			<dl>
-				<dt>Objective</dt><dd>Users have full control of major content changes</dd>
-				<dt>Author task</dt><dd>Provide ways for users to trigger or turn off changes of context</dd>
-			</dl>
+         <dl>
+		         <dt>Goal</dt><dd>Users have full control of major content changes.</dd>
+		         <dt>What to do</dt><dd>Provide ways for users to trigger or turn off changes of context.</dd>
+			      <dt>Why it's important</dt><dd>Content that behaves predictably is especially important to people with disabilities.</dd> 
```

Comment:
@GreggVan suggested
```suggestion
			      <dt>Why it's important</dt><dd>Content that behaves predictably is especially important to people with disabilities as is ability to keep content from changing while they are trying to view it</dd> 
```

Reply:
@GreggVan also suggested
```suggestion
			      <dt>Why it's important</dt><dd>Content that behaves predictably is especially important to people with disabilities and content that changes above their point of reading while it is being read is difficult for some people, and completely missed by others with low vision or using a screen reader.</dd> 
```

Reply:
In my opinion, both of these suggestions are getting into too much detail for a brief description. As well, some of the wording here, such as changes above the point of reading, are not actually mentioned in the understanding document.
If it is felt they should be included, a PR should be created for the Understanding document.